config/routes.rb

```diff
@@ -5,6 +5,7 @@
 
   root to: 'homepage#index', via: :get
   get "/homepage" => redirect("/")
+  match "/", to: proc { |env| [405, {}, ["Not Allowed"]] }, via: :all
```

Comment:
Is this bit necessary, if we already have the `match "*any"`?

Reply:
The `any*` doesn't seem to match `/` unfortunately